src/main/java/org/folio/circulation/services/LostItemFeeRefundService.java

```diff
@@ -203,8 +210,47 @@ private static boolean shouldSchedulePatronNotices(LostItemFeeRefundContext cont
     final Result<CqlQuery> fetchQuery = exactMatch("loanId", context.getLoan().getId())
       .combine(exactMatchAny("feeFineType", feeFineTypes), CqlQuery::and);
 
-    return accountRepository.findAccountsAndActionsForLoanByQuery(fetchQuery)
-      .thenApply(r -> r.map(context::withAccounts));
+      return accountRepository.findAccountsAndActionsForLoanByQuery(fetchQuery)
+        .thenApply(r -> r.map(this::filterAccountsForRefund))
```

Comment:
Could we replace 'r' into something more human readable?

Reply:
@viktor-draban this is a common approach used throughout Folio